### PR TITLE
[Modal Layout Picker] Show toast messages after creating a new page

### PR DESCRIPTION
**Pull request by antonis**
**Fixes:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2453

**Related PRs:** 
Depends on: https://github.com/wordpress-mobile/WordPress-Android/pull/13077
`gutenberg-mobile` https://github.com/wordpress-mobile/gutenberg-mobile/pull/2697
`gutenberg` https://github.com/WordPress/gutenberg/pull/25951

To test:

Layout Picker should show [when creating a new page from *My Site* or *Site Pages* the *Modal Layout Picker* appears](https://github.com/wordpress-mobile/WordPress-Android/pull/12482).

- Start by navigating to the Modal Layout Picker

### Page Created (with content)
1. Select a Layout from the options in Modal Layout Picker
1. Select "Create Page"
**Expect** to see the info notice banner appear under the navigation bar with the text "Page created"
- 🗒️ This is different than the mocks in the linked issue but aligns with the direction that was determined. 

### Blank Page Created
1. Select "Create Blank Page"
**Expect** to see the info notice banner appear under the navigation bar with the text "Blank page created"
- 🗒️ This is different than the mocks in the linked issue but aligns with the direction that was determined. 

### Blog Post Created
1. Create a new blog post
- This should not display the notice bar

### Open existing Page Created
1. Open an existing page (either published or draft)
- This should not display the notice bar

## Screenshots 

|Blank page|Page with layout|
|---|---|
|device-2020-10-08-152140|device-2020-10-08-152217|

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by Tug — APPROVED** [review]
LGTM 👍

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java [hunk 1]
```diff
@@ -647,6 +647,18 @@ protected void onCreate(Bundle savedInstanceState) {
         setupPreviewUI();
     }
 
+    private void presentNewPageNoticeIfNeeded() {
+        if (mIsPreview
+            || !mIsPage
+            || !mModalLayoutPickerFeatureConfig.isEnabled()
+            || !mEditPostRepository.isLocalDraft()) {
```
Comment (malinajirka) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java]: Could you please double check we really want to use `!mEditPostRepository.isLocalDraft()` here? `Local draft` means that the post is a draft and has not been synced with the server yet. I tried to reproduce an issue with this condition and I think I was successful:

1. Click on Create a Page
2. Select a layout
3. Click on "Create Page"
4. Notice "Page Created" message (this is expected)
5. Turn on airplane mode
6. Leave the editor
7. Go to "Page list"
8. Open the page from step 3
9. Notice" Page created" message (this is imo not expect)


Perhaps the `mIsNewPost` field in the EditPostActivity might work for us. Wdyt?

**Review by malinajirka — COMMENTED** [review]
Thanks @antonis! I haven't tested the app (except of one specific scenario), since TUG has already tested it. I found one potential issue, let me know what you think.

File: WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java (hunk 1 above)
Comment (antonis) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java]: Thank you for reviewing this @malinajirka 🙇 
Nice catch 👍 
Fixed with 80ba44fe8dd2945acb477f84c1bc487eef7510c5

**Review by malinajirka — APPROVED** [review]
LGTM, thanks!

**Pull request merged**